package.json

```diff
@@ -44,29 +44,29 @@
     "minimist": "1.2.0",
     "postcss-loader": "^2.0.5",
     "prompt": "1.0.0",
-    "react-dev-utils": "^3.1.1",
-    "react-error-overlay": "^1.0.10",
+    "react-dev-utils": "^4.0.1",
+    "react-error-overlay": "^2.0.1",
     "string-replace-loader": "^1.3.0",
     "style-loader": "^0.18.1",
     "sw-precache-webpack-plugin": "^0.11.4",
     "url-loader": "^0.5.9",
-    "webpack": "^3.5.5",
+    "webpack": "^3.5.6",
     "webpack-dev-server": "^2.7.1"
   },
   "devDependencies": {
     "babel-eslint": "^7.2.3",
-    "chai": "^4.1.0",
     "commitizen": "^2.9.6",
     "cz-conventional-changelog": "^2.0.0",
     "dir-compare": "^1.3.0",
-    "eslint": "^4.3.0",
-    "eslint-plugin-prettier": "^2.1.2",
+    "eslint": "^4.6.1",
+    "eslint-plugin-prettier": "^2.2.0",
     "husky": "^0.14.3",
-    "mocha": "^3.5.0",
+    "mocha": "^3.5.2",
     "nightmare": "^2.10.0",
-    "prettier": "^1.5.3",
+    "prettier": "^1.6.1",
     "rimraf": "^2.5.4",
-    "semantic-release": "^6.3.2"
+    "semantic-release": "^7.0.2",
+    "unexpected": "^10.35.0"
```

Comment:
For what reason you have changed `chai` to `unexpected`?

Reply:
@fobos it produces more readable test errors, obviously, we don't have that many tests, so it's a little smaller as well.

I would roll back if you have a strong opinion on assertion lib. I've been doing a lot of testing with unexpected recently and I would say I've had a better time :slightly_smiling_face:

Reply:
Thanks for explanation. 

> I would roll back if you have a strong opinion on assertion lib.
It's ok to use `unexpected`. I see your points.